sample-apps/java-lambda/README.md

```diff
@@ -0,0 +1,132 @@
+![CI](https://github.com/aws-observability/aws-otel-lambda/workflows/CI/badge.svg)
+# AWS Distro for OpenTelemetry Lambda Support (Metrics) 
+The AWS Distro for OpenTelemetry(ADOT) [Lambda layer](https://docs.aws.amazon.com/lambda/latest/dg/configuration-layers.html) provides a simple user experience by attaching an additional layer to Lambda function. Users can onload and offload OpenTelemetry from their Lambda function with minimal code change.
+
+
+## Sample
+This sample will create AWS Distro for OpenTelemetry Lambda Extension and a sample application in region us-west-2 in your AWS account. By default the metrics are sent to AWS CloudWatch, you can see telemetry data generated by OpenTelemetry from AWS CloudWatch Console.
+- Install [SAM CLI](https://docs.aws.amazon.com/serverless-application-model/latest/developerguide/serverless-sam-cli-install.html) and [AWS CLI](https://docs.aws.amazon.com/cli/latest/userguide/install-cliv2.html). For users have trouble in SAM CLI, see [documentation](docs/misc/sam.md).
```

Comment:
It seems docs/miscs/sam.md is not in this PR? python one does have the misc.md file, can we move the common doc into a shared folder or change the link here to refer to the doc in python sample.